### ISSUE TITLE
Loss converges to 12.5 and GPU memory oscillating

### ISSUE THREAD

**Issue by michaelfuchs1** [qqwweee/keras-yolo3#143]
Dear all, 

Thanks in advance for you reply !

I am currently training this keras implementation on my own data set with one class (trying to detect a tennis ball during a game on TV). I use 6039 labelled pictures. Batch size 32. Anchors unchanged. So far I only managed to train it with the last 3 layers unfrozen, not yet the full model. 

During training 2 things surprise me: 
- my loss value doesn't decrease towards a small value as it normally would with other NN. for example on yolov2 from loss converges towards 0.05. Here it seems not to go below 12. Is this normal ?
- my GPU memory (eGPU GTX1080) is constantly oscillating between 0% and around 45% during training. is this normal ? should I change certain hyperparameters to ensure better usage ?
![gpu](https://user-images.githubusercontent.com/22610804/42206683-2589f1b2-7ea8-11e8-8822-ee1e6cf549ff.JPG)

Again thanks for your inputs !

**Comment by jinxie0731** [comment]
Did you use the same set of labelled pictures when you got the loss of about 0.05 from Yolov2?
I also got a loss value of about 12 when I trained my own model on VOC data set. Batch size is 10. Anchors unchanged but classes is 12. I trained it with last 3 layers unfrozen for 50 epochs first, and then unfrozen all layers and trained another 50 epochs.

**Comment by michaelfuchs1** [comment]
Thanks for your reply !

Yes, same sort of data set for yolo2 and yolo3. 
My loss goes up and down between 12.7 and 12.8 from epochs 38 until 100 with frozen layers. 
I will try this week to unfreeze the other layers. 

Do you have the same behavior for your GPU memory ?
